packages/http-client-csharp/emitter/test/Unit/input-parameter.test.ts

```diff
@@ -160,10 +160,10 @@ describe("Test Parameter Explode", () => {
         const context = createEmitterContext(program);
         const sdkContext = await createCSharpSdkContext(context);
         const root = createModel(sdkContext);
-        const inputParamArray = root.Clients[0].Operations[0].Parameters.filter(
+        const inputParamArray = root.Clients[0].operations[0].Parameters.filter(
```

Comment:
Is this casing change intended?

Reply:
yeah. Everything comes from tcgc has properties with lower cases. I think we probably need to change all the property names to start with lower cases.